cypress/integration/activair_spec.js

```diff
@@ -0,0 +1,49 @@
+import key from '../../src/keyAPI'
+
+describe('ActivAir Website', () => {
+  const baseUrl = 'http://localhost:8080/'
+  const citiesAPI = `http://api.airvisual.com/v2/cities?state=Colorado&country=USA&key=${key}`
+  const statesAPI = `http://api.airvisual.com/v2/states?country=USA&key=${key}`
+  const cityStateAPI = `http://api.airvisual.com/v2/city?city=Denver&state=Colorado&country=USA&key=${key}`
+  const nearestAPI = `http://api.airvisual.com/v2/nearest_city?key=${key}`
+
+  it('Should display a header with website name and subheader', () => {
+    cy.intercept('GET', statesAPI, { fixture: 'state-data' })
+      .visit(baseUrl)
+      .get('header')
+      .get('h1').should('have.text', 'ActivAir')
+      .get('h2').should('have.text', 'Find fresh air for your outdoor activities.')
+  })
+
+  it('Should populate a location card with mock nearest location data', () => {
+    cy.intercept('GET', nearestAPI, { fixture: 'location-thornton' })
+      .intercept('GET', statesAPI, { fixture: 'state-data' })
+      .visit(baseUrl)
+      .get('input[name=nearestLocation]').click()
+      .get('.location-card').find('h2').should('have.text', 'Thornton, Colorado')
+  })
+
+  it('Should populate state and city dropdowns with relevant information and create a card from the selected location', () => {
+    cy.intercept('GET', cityStateAPI, { fixture: 'location-denver' })
+      .intercept('GET', statesAPI, { fixture: 'state-data' })
+      .intercept('GET', citiesAPI, { fixture: 'city-data' })
+      .visit(baseUrl)
+      .get('select#dropdown-states').select('Colorado').should('have.value', 'Colorado')
+      .get('select#dropdown-cities').select('Denver').should('have.value', 'Denver')
+      .get('input[name=submit]').click()
+      .get('.location-card').find('h2').should('have.text', 'Denver, Colorado')
+      .get('.location-card').find('.button-delete').click()
```

Comment:
It might be more thorough to add a `.get('.location-card').should('not.contain', 'Denver, Colorado')` after the delete button has been clicked.

Reply:
If we did so, then this if block would technically be testing 3 different things. Not sure if it would be best practice to split them up to be SRP at that point, even if it is a bit redundant.

Reply:
I don't think we can check whether the location card has text after it's been deleted. We may be able to test whether the parent element has any cards? idk which direction we want to take

Reply:
I think you can with the `.should('not.contain'` bit? That's how we tested it in Tomatillos!